badge.html

```diff
@@ -44,22 +49,26 @@ <h1 class="lib-header"> uiStore </h1>
+           </p>
             <div>
                 <span class="badge badge-primary"> primary </span>
                 <span class="badge badge-secondary"> secondary </span>
                 <span class="badge badge-success"> success </span>
                 <span class="badge badge-failure"> failure </span>
                 <span class="badge badge-warning"> warning </span>
                 <span class="badge badge-info"> info </span>
-                <span class="badge badge-special"> info </span>
-
+                <span class="badge badge-special"> popular </span>
+                
             </div>
-            <iframe
-src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=hopscotch&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%253E%250A%2520%2520%2520%253Cspan%2520class%253D%2522badge%2520badge-primary%2522%253E%2520primary%2520%253C%252Fspan%253E%250A%2520%2520%2520%253Cspan%2520class%253D%2522badge%2520badge-secondary%2522%253E%2520secondary%2520%253C%252Fspan%253E%250A%2520%2520%2520%253Cspan%2520class%253D%2522badge%2520badge-success%2522%253E%2520success%2520%253C%252Fspan%253E%250A%2520%2520%2520%253Cspan%2520class%253D%2522badge%2520badge-failure%2522%253E%2520failure%2520%253C%252Fspan%253E%250A%2520%2520%2520%253Cspan%2520class%253D%2522badge%2520badge-warning%2522%253E%2520warning%2520%253C%252Fspan%253E%250A%2520%2520%2520%253Cspan%2520class%253D%2522badge%2520badge-info%2522%253E%2520info%2520%253C%252Fspan%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253C%252Fdiv%253E"
-style="width: 673px; height: 329px; border:0; transform: scale(1); overflow:hidden;"
-sandbox="allow-scripts allow-same-origin">
-</iframe>
+            
         </div> <!--end of class badges-->
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=blackboard&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cspan%2520class%253D%2522badge%2520badge-primary%2522%253E%2520primary%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-secondary%2522%253E%2520secondary%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-success%2522%253E%2520success%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-failure%2522%253E%2520failure%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-warning%2522%253E%2520warning%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-info%2522%253E%2520info%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-special%2522%253E%2520popular%2520%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+  style="border:0; transform: scale(1); overflow:hidden;"
```

Comment:
in iframes the inline style which is getting repeated can be avoided by adding them to a class

Reply:
Done